src/mds/CInode.h

```diff
@@ -748,18 +790,21 @@ class CInode : public MDSCacheObject, public InodeStore {
   bool is_any_caps() { return !client_caps.empty(); }
   bool is_any_nonstale_caps() { return count_nonstale_caps(); }
 
+  const std::map<int32_t,int32_t>& get_mds_caps_wanted() const { return mds_caps_wanted; }
   std::map<int32_t,int32_t>& get_mds_caps_wanted() { return mds_caps_wanted; }
 
-  std::map<client_t,Capability*>& get_client_caps() { return client_caps; }
+  const std::map<client_t,Capability*>& get_client_caps() const { return client_caps; }
   Capability *get_client_cap(client_t client) {
     if (client_caps.count(client))
       return client_caps[client];
     return 0;
   }
-  int get_client_cap_pending(client_t client) {
-    Capability *c = get_client_cap(client);
-    if (c) return c->pending();
-    return 0;
+  int get_client_cap_pending(client_t client) const {
+    if (client_caps.count(client)) {
+      return client_caps.find(client)->second->pending();
```

Comment:
It's a silly little thing, but there are some static analyzers that will complain about our not using iterators here — makes life harder on the compiler when optimizing.